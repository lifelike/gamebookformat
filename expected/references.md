# Gamebook #


## Introduction ##
 This gamebook demonstrates simple references between sections. Also notice that an intro section like this one can reference sections, like the start at [1](#section1)
 or the next section at [400](#section400)
. References can include an optional name that is used in some formats, so the same links could also be rendered as [start](#section1)
 and [next](#section400)
. 

Turn to 1 to begin.
<a name="section1">

## 1 ##
 This is where the adventure begins. You can go on to the next section, see [400](#section400)
 or try the random selection feature at [399](#section399)
. 
<a name="section397">

## 397 ##
 The end. 
<a name="section398">

## 398 ##
 Alternative ending. 
<a name="section399">

## 399 ##
 Pick a destination at random from [400](#section400)
 or [397](#section397)
 or [398](#section398)
. 
<a name="section400">

## 400 ##
 This is the next section. Go on to the end at [397](#section397)
. 

