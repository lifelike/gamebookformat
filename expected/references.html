<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta name="viewport"
        content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>Gamebook</title>
    <script type="text/javascript" src="gamebookformatplay.js"></script>
    <link rel="stylesheet" href="gamebookformat.css"
          type="text/css" />
    <script>
      if (typeof gamebook !== 'undefined') {
         gamebook.id = 'references';
      }
    </script>
  </head>
  <body>
<div class="hideintrolink nodisplay"
    onclick="gamebook.hideIntroSections()">(hide instructions)</div>
  <div class="gamebook">
<div class="introsection">
<div class="introsectionheading">Introduction</div>
<div class="introsectionbody">
 This gamebook demonstrates simple references between sections. Also notice that an intro section like this one can reference sections, like the start at <a class="sectionref enabledlink" data-ref="1"
href="#section1">1</a> or the next section at <a class="sectionref enabledlink" data-ref="400"
href="#section400">400</a>. References can include an optional name that is used in some formats, so the same links could also be rendered as <a class="sectionref enabledlink" data-ref="1"
href="#section1">start</a> and <a class="sectionref enabledlink" data-ref="400"
href="#section400">next</a>. 
</div>
</div>
<div class="resumelink nodisplay"
  onclick="gamebook.loadGame()">Resume saved game.</div>

<div class="startlink"
  onclick="gamebook.turnTo(1)">Turn to 1 to begin.</div>
<script>
    if (typeof gamebook !== 'undefined' && gamebook.hasSavedGame()) {
        var resumeLinks = document.getElementsByClassName('resumelink');
        Array.prototype.forEach.call(resumeLinks, function(e) {
            e.classList.remove('nodisplay');
        });
    }
</script>
<div class="section" id="section1"
     >
<div class="sectionnumber" id="para1">1</div>
<div class="sectiontext">
 This is where the adventure begins. You can go on to the next section, see <a class="sectionref enabledlink" data-ref="400"
href="#section400">400</a> or try the random selection feature at <a class="sectionref enabledlink" data-ref="399"
href="#section399">399</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(1, document.getElementById('section1'));
}
</script>
<div class="section" id="section397"
     >
<div class="sectionnumber" id="para397">397</div>
<div class="sectiontext">
 The end. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(397, document.getElementById('section397'));
}
</script>
<div class="section" id="section398"
     >
<div class="sectionnumber" id="para398">398</div>
<div class="sectiontext">
 Alternative ending. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(398, document.getElementById('section398'));
}
</script>
<div class="section" id="section399"
     >
<div class="sectionnumber" id="para399">399</div>
<div class="sectiontext">
 <span class="random enabledlink">Pick a destination at random</span> from <a class="sectionref enabledlink" data-ref="400"
href="#section400">400</a> or <a class="sectionref enabledlink" data-ref="397"
href="#section397">397</a> or <a class="sectionref enabledlink" data-ref="398"
href="#section398">398</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(399, document.getElementById('section399'));
}
</script>
<div class="section" id="section400"
     >
<div class="sectionnumber" id="para400">400</div>
<div class="sectiontext">
 This is the next section. Go on to the end at <a class="sectionref enabledlink" data-ref="397"
href="#section397">397</a>. 
</div>
</div>
<script>
if (typeof gamebook !== 'undefined') {
   gamebook.addSection(400, document.getElementById('section400'));
}
</script>
    <div id="counters" class="counters">
    </div>
    <div id="counterTemplate" class="counterTemplate">
      <span class="counterheading"></span>
      <span class="countercontents"></span>
    </div>
    </div>
    <div id="collections" class="collections">
    </div>
    <div id="collectionTemplate" class="collectionTemplate">
      <span class="collectionheading"></span>
      <span class="collectioncontents"></span>
    </div>
    </div>
    <script>
if (typeof gamebook !== 'undefined') {
   gamebook.prepare();
}
    </script>
<div class="displayintrolink nodisplay"
    onclick="gamebook.showIntroSections()">(show instructions)</div>
    </div>
  </body>
</html>
